<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>EmailDialogBase</class>
 <widget class="QDialog" name="EmailDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string comment="dialog header" >Email List</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="addEmailBtn" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text" >
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeEmailBtn" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="emailUpBtn" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="emailDownBtn" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="emailList" />
   </item>
   <item>
    <widget class="QLineEdit" name="emailEdit" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addEmailBtn</tabstop>
  <tabstop>removeEmailBtn</tabstop>
  <tabstop>emailUpBtn</tabstop>
  <tabstop>emailDownBtn</tabstop>
  <tabstop>emailList</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
